docs/DeveloperGuide.adoc

```diff
@@ -836,6 +836,58 @@ Use case ends.
 Use case resumes at step 2.
 
 [discrete]
+=== Use case: Undo last command
+
+*MSS*
+
+1. User requests to undo last command
+2. AddressBook confirms if user really wishes to undo last command
+3. User confirms the undo
+4. AddressBook undoes the last command
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. User cancels the undo.
++
+Use case ends.
+
+* 4a. The last command is undoable (i.e. a command that does not modify AddressBook's content)
```

Comment:
Do you mean: the last command is *not* undoable?

In that case, should we just record undoable commands instead? Cause this implies that if the user performs an non-undoable command, then they can't undo anymore (cause the last command is always non-undoable).

Reply:
Ah, you're 100% right in saying that I meant the last command is *not* undoable. 

And yes, that is a suggestion which I think we should implement as well. I will look into this again, thanks for the feedback!